### ISSUE TITLE
Sam's review of Brandon's poster

### ISSUE THREAD

**Issue by samm82** [JacquesCarette/Drasil#3921]
Here are my (rough) notes on your (so far, very good!) poster - feel free to pull any of these out to separate issues if they'll require more discussion

- [x] I would explicitly define what you mean by "shared" earlier (i.e., in the Goal section)
- [x] What's the significance of "OO typeclasses can inherit from Shared typeclasses."? (this may be addressed by the above point)
- [x] What do outline colours mean in your graphs?
- [x] Captions should be sentence case and end with a period

Minor things:
- [x] "5" should be spelled out in the Introduction
- [x] "E.g." shouldn't start a sentence; use "For example" instead
- [x] I'd add a space before the line colour note and re- add the "Legend" heading, and I don't think the parentheses are needed
- [x] Explicitly explaining that OO = object- oriented might help? idk if this is assumed knowledge
- [x] Is there a reason "Scientific Computing" is capitalized?
- [x] Could you use the colours from the Julia logo in your graphs? That might look cool

**Comment by B-rando1** [comment]
> Explicitly explaining that OO = object- oriented might help? idk if this is assumed knowledge

For this, would it be enough to say 'Object-Oriented (OO)' the first time the term comes up?

**Comment by B-rando1** [comment]
> What do outline colours mean in your graphs?

The outline colours are an artifact from the original graph that I based these on.  They mean the same things as the line colours.  Do you think I should make all the outlines black since we have the lines?

**Comment by B-rando1** [comment]
> I would explicitly define what you mean by "shared" earlier (i.e., in the Goal section)
> What's the significance of "OO typeclasses can inherit from Shared typeclasses."? (this may be addressed by the above point)

Is this what you were thinking?
![image](https://github.com/user-attachments/assets/ebdb170b-1d56-45ab-9b57-72d63c32cdc0)

**Comment by samm82** [comment]
Nice! (I'd also make "split" lowercase in the Solution bullet point, just because I like details 🙃)

**Comment by B-rando1** [comment]
That was a cool idea to use the Julia colours!  I tried changing them, this is what I got:
![image](https://github.com/user-attachments/assets/ca4f2347-4528-4dfd-a0b5-e62d7b22a94e)
My one concern is that the contrast might not be as high.  Do you think it's still readable enough?

**Comment by smiths** [comment]
@B-rando1 the graph above is great, but it is hard to read.  The previous version was also hard to read.  I think you have room to make the text in each of the boxes larger.

**Comment by smiths** [comment]
@B-rando1 some quick feedback:

- I suggest that you start with your goal (add Julia to GOOL) and then say what GOOL is, what the problem is, and your proposed solution.  Your poster starts with background information currently.  It is good to start with the main point of the poster, since this is the first thing (and sometimes the only thing) that a passerby will read.
- the pie charts are nice!
- can you quantify how many lines of Julia are now generated?

**Comment by B-rando1** [comment]
@samm82 @smiths thanks for the helpful feedback!  I made changes based on it, and updated the remote [file](https://github.com/JacquesCarette/Drasil/blob/Poster/People/Brandon-Bosman/WIP%20Poster.pdf).

A couple notes:
- I made the font size of the main diagram a little bigger.  It's still pretty small, but it's better.  I might see if I can make it bigger yet without having to move stuff around too much.
- Using the `wc -l` command with `find`, I got that there are 545 lines of Julia code being generated (excluding the blank line at the bottom of each file).  It looks like about half of them are from our GOOL tests, and half of them are from the two versions of Projectile that the Julia renderer supports.
